pubnub/pubnub_asyncio.py

```diff
@@ -179,7 +179,7 @@ async def _request_helper(self, options_func, cancellation_event):
             url = utils.build_url(scheme="", origin="", path=options.path, params=options.query_string)
 
         url = URL(url, encoded=True)
-
+        print(f'\033[48;5;236mEndpoint Subscribe async, {url}\033[0m\n')
```

Comment:
Is it OK to leave this print statement? Don't you have any logger instance that covers raw prints?

Reply:
fixed. commited debug by mistake